common/lib/capa/capa/templates/codeinput.html

```diff
@@ -1,9 +1,11 @@
 <%! from django.utils.translation import ugettext as _ %>
 <section id="textbox_${id}" class="capa_inputtype textbox cminput">
-  <textarea rows="${rows}" cols="${cols}" name="input_${id}" 
-            aria-label="${_("{programming_language} editor").format(programming_language=mode)}" 
```

Comment:
Both textarea's need a label if we want to avoid automatic test failures.  Even though the user doesn't interact directly with this one, I think we should leave the aria-label there.  Is there a particular reason why you want to remove it?